### PR TITLE
OCM-9301 | test: updated case id:68173

### PR DESCRIPTION
**Pull request by patriksuba**
Update naming for Additional Security Group IDs in NodePoolDescription structure

`ginkgo run -focus 68173 tests/e2e
Ginkgo detected a version mismatch between the Ginkgo CLI and the version of Ginkgo imported by your packages:
  Ginkgo CLI Version:
    2.14.0
  Mismatched package versions found:
    2.17.1 used by e2e

  Ginkgo will continue to attempt to run but you may see errors (including flag
  parsing errors) and should either update your go.mod or your version of the
  Ginkgo CLI to match.

  To install the matching version of the CLI run
    go install github.com/onsi/ginkgo/v2/ginkgo
  from a path that contains a go.mod file.  Alternatively you can use
    go run github.com/onsi/ginkgo/v2/ginkgo
  from a path that contains a go.mod file to invoke the matching version of the
  Ginkgo CLI.

  If you are attempting to test multiple packages that each have a different
  version of the Ginkgo library with a single Ginkgo CLI that is currently
  unsupported.
  
Running Suite: ROSA CLI e2e tests suite - /home/psuba/workspace/rosa/tests/e2e
==============================================================================
Random Seed: 1719930662

Will run 1 of 127 specs
SSSSSSSStime="2024-07-02 16:31:12" level=info msg="Trying to list subnets of the vpc"
time="2024-07-02 16:31:12" level=info msg="Got 6 subnets"
time="2024-07-02 16:31:12" level=info msg="subnet-0ef2a8aadcb0a7e5e\t10.0.3.0/24\tus-west-2b\t"
time="2024-07-02 16:31:12" level=info msg="subnet-099c46af25323f9ae\t10.0.4.0/24\tus-west-2c\t"
time="2024-07-02 16:31:12" level=info msg="subnet-03b9dd152cf86210f\t10.0.5.0/24\tus-west-2c\t"
time="2024-07-02 16:31:12" level=info msg="subnet-07b6cf81d4f4a3e13\t10.0.2.0/24\tus-west-2b\t"
time="2024-07-02 16:31:12" level=info msg="subnet-04882c25c7b78c6d0\t10.0.1.0/24\tus-west-2a\t"
time="2024-07-02 16:31:12" level=info msg="subnet-0ec3c286a3255b6c7\t10.0.0.0/24\tus-west-2a\t"
time="2024-07-02 16:31:12" level=info msg="Got main association for rt rtb-03a3abc97a7be3f91"
time="2024-07-02 16:31:13" level=info msg="Create security group sg-0aeef5d99c414a9d6 success for vpc-005c008afbc02c569"
time="2024-07-02 16:31:13" level=info msg="Tag resource sg-0aeef5d99c414a9d6 successfully"
time="2024-07-02 16:31:13" level=info msg="Created tagged security group with ID sg-0aeef5d99c414a9d6"
time="2024-07-02 16:31:14" level=info msg="Authorize security group success sg-0aeef5d99c414a9d6"
time="2024-07-02 16:31:14" level=info msg="Authorize security group success sg-0aeef5d99c414a9d6"
time="2024-07-02 16:31:15" level=info msg="Create security group sg-0e47b787d1720f697 success for vpc-005c008afbc02c569"
time="2024-07-02 16:31:15" level=info msg="Tag resource sg-0e47b787d1720f697 successfully"
time="2024-07-02 16:31:15" level=info msg="Created tagged security group with ID sg-0e47b787d1720f697"
time="2024-07-02 16:31:16" level=info msg="Authorize security group success sg-0e47b787d1720f697"
time="2024-07-02 16:31:16" level=info msg="Authorize security group success sg-0e47b787d1720f697"
time="2024-07-02 16:31:16" level=info msg="Create security group sg-05186b23ddf894d5f success for vpc-005c008afbc02c569"
time="2024-07-02 16:31:17" level=info msg="Tag resource sg-05186b23ddf894d5f successfully"
time="2024-07-02 16:31:17" level=info msg="Created tagged security group with ID sg-05186b23ddf894d5f"
time="2024-07-02 16:31:17" level=info msg="Authorize security group success sg-05186b23ddf894d5f"
time="2024-07-02 16:31:18" level=info msg="Authorize security group success sg-05186b23ddf894d5f"
time="2024-07-02 16:32:00" level=info msg="Release rules successfully for SG sg-0aeef5d99c414a9d6"
time="2024-07-02 16:32:01" level=info msg="Delete security group sg-0aeef5d99c414a9d6 success "
time="2024-07-02 16:32:02" level=info msg="Release rules successfully for SG sg-0e47b787d1720f697"
time="2024-07-02 16:32:02" level=info msg="Delete security group sg-0e47b787d1720f697 success "
time="2024-07-02 16:32:03" level=info msg="Release rules successfully for SG sg-05186b23ddf894d5f"
time="2024-07-02 16:32:04" level=info msg="Delete security group sg-05186b23ddf894d5f success "
•SSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSS

Ran 1 of 127 Specs in 61.074 seconds
SUCCESS! -- 1 Passed | 0 Failed | 0 Pending | 126 Skipped
PASS

Ginkgo ran 1 suite in 1m1.960459134s
Test Suite Passed
`

### CODE REVIEW

**Comment by yasun1** [comment]
@patriksuba Please follow up the [guideline](https://github.com/openshift/rosa/tree/master/tests#contribute-to-day2) to define your commit title.

File: tests/utils/exec/rosacli/machinepool_service.go [hunk 1]
```diff
@@ -121,7 +121,7 @@ type NodePoolDescription struct {
 	ManagementUpgrade          []map[string]string `yaml:"Management upgrade,omitempty"`
 	Message                    string              `yaml:"Message,omitempty"`
 	ScheduledUpgrade           string              `yaml:"Scheduled upgrade,omitempty"`
-	AdditionalSecurityGroupIDs string              `yaml:"Additional security group IDs,omitempty"`
+	AdditionalSecurityGroupIDs string              `yaml:"Additional Security Group IDs,omitempty"`
```
Comment (xueli181114) [inline tests/utils/exec/rosacli/machinepool_service.go:124]: No, it's not right to fix here.

Comment (xueli181114) [reply tests/utils/exec/rosacli/machinepool_service.go:124]: Fix `MachinePoolDescription` struct
And update the reflect function

**Comment by patriksuba** [comment]
Update naming for Additional Security Group IDs in MachinePoolDescription structure

`ginkgo run -focus 68173 tests/e2e
Ginkgo detected a version mismatch between the Ginkgo CLI and the version of Ginkgo imported by your packages:
  Ginkgo CLI Version:
    2.14.0
  Mismatched package versions found:
    2.17.1 used by e2e

  Ginkgo will continue to attempt to run but you may see errors (including flag
  parsing errors) and should either update your go.mod or your version of the
  Ginkgo CLI to match.

  To install the matching version of the CLI run
    go install github.com/onsi/ginkgo/v2/ginkgo
  from a path that contains a go.mod file.  Alternatively you can use
    go run github.com/onsi/ginkgo/v2/ginkgo
  from a path that contains a go.mod file to invoke the matching version of the
  Ginkgo CLI.

  If you are attempting to test multiple packages that each have a different
  version of the Ginkgo library with a single Ginkgo CLI that is currently
  unsupported.
  
Running Suite: ROSA CLI e2e tests suite - /home/psuba/workspace/rosa/tests/e2e
==============================================================================
Random Seed: 1720008293

Will run 1 of 127 specs
SSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSStime="2024-07-03 14:05:02" level=info msg="Trying to list subnets of the vpc"
time="2024-07-03 14:05:02" level=info msg="Got 6 subnets"
time="2024-07-03 14:05:02" level=info msg="subnet-06359cc1ba909144b\t10.0.3.0/24\tus-east-1b\t"
time="2024-07-03 14:05:02" level=info msg="subnet-03db0d17b6b0d9054\t10.0.4.0/24\tus-east-1c\t"
time="2024-07-03 14:05:02" level=info msg="subnet-05658293263a933bd\t10.0.5.0/24\tus-east-1c\t"
time="2024-07-03 14:05:02" level=info msg="subnet-02a1a61476f695b01\t10.0.0.0/24\tus-east-1a\t"
time="2024-07-03 14:05:02" level=info msg="subnet-06bb8559c53483a10\t10.0.2.0/24\tus-east-1b\t"
time="2024-07-03 14:05:02" level=info msg="subnet-0449e3eea57b64f52\t10.0.1.0/24\tus-east-1a\t"
time="2024-07-03 14:05:02" level=info msg="Got custom rt rtb-0e29314bfea3bad14 "
time="2024-07-03 14:05:02" level=info msg="Got custom rt rtb-0c013cf981d203f84 "
time="2024-07-03 14:05:02" level=info msg="Got custom rt rtb-0c4b81cdf1df9a177 "
time="2024-07-03 14:05:02" level=info msg="Got custom rt rtb-01d2c22690058b9da "
time="2024-07-03 14:05:02" level=info msg="Got main association for rt rtb-0406ea4480d8760e3"
time="2024-07-03 14:05:02" level=info msg="Got custom rt rtb-0a2e723b8d5dc22de "
time="2024-07-03 14:05:02" level=info msg="Got custom rt rtb-017ba4e46164b7c97 "
time="2024-07-03 14:05:03" level=info msg="Create security group sg-041be66429e0d4125 success for vpc-0be53849de9dce4e5"
time="2024-07-03 14:05:04" level=info msg="Tag resource sg-041be66429e0d4125 successfully"
time="2024-07-03 14:05:04" level=info msg="Created tagged security group with ID sg-041be66429e0d4125"
time="2024-07-03 14:05:04" level=info msg="Authorize security group success sg-041be66429e0d4125"
time="2024-07-03 14:05:05" level=info msg="Authorize security group success sg-041be66429e0d4125"
time="2024-07-03 14:05:06" level=info msg="Create security group sg-07cb37f4d98e55bf3 success for vpc-0be53849de9dce4e5"
time="2024-07-03 14:05:07" level=info msg="Tag resource sg-07cb37f4d98e55bf3 successfully"
time="2024-07-03 14:05:07" level=info msg="Created tagged security group with ID sg-07cb37f4d98e55bf3"
time="2024-07-03 14:05:08" level=info msg="Authorize security group success sg-07cb37f4d98e55bf3"
time="2024-07-03 14:05:09" level=info msg="Authorize security group success sg-07cb37f4d98e55bf3"
time="2024-07-03 14:05:10" level=info msg="Create security group sg-082a34d537d9b7895 success for vpc-0be53849de9dce4e5"
time="2024-07-03 14:05:10" level=info msg="Tag resource sg-082a34d537d9b7895 successfully"
time="2024-07-03 14:05:10" level=info msg="Created tagged security group with ID sg-082a34d537d9b7895"
time="2024-07-03 14:05:11" level=info msg="Authorize security group success sg-082a34d537d9b7895"
time="2024-07-03 14:05:12" level=info msg="Authorize security group success sg-082a34d537d9b7895"
time="2024-07-03 14:05:49" level=info msg="Release rules successfully for SG sg-041be66429e0d4125"
time="2024-07-03 14:05:49" level=info msg="Delete security group sg-041be66429e0d4125 success "
time="2024-07-03 14:05:50" level=info msg="Release rules successfully for SG sg-07cb37f4d98e55bf3"
time="2024-07-03 14:05:50" level=info msg="Delete security group sg-07cb37f4d98e55bf3 success "
time="2024-07-03 14:05:51" level=info msg="Release rules successfully for SG sg-082a34d537d9b7895"
time="2024-07-03 14:05:51" level=info msg="Delete security group sg-082a34d537d9b7895 success "
•SSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSS

Ran 1 of 127 Specs in 57.109 seconds
SUCCESS! -- 1 Passed | 0 Failed | 0 Pending | 126 Skipped
PASS

Ginkgo ran 1 suite in 57.992865953s
Test Suite Passed
`

**Pull request merged**